daemon/includes/log.h

```diff
@@ -21,10 +21,18 @@
 #if defined(__TIZEN__)
 #include <dlog.h>
 
-#define _D(fmt, arg...)		do { SLOGD(fmt, ##arg); } while (0)
-#define _I(fmt, arg...)		do { SLOGI(fmt, ##arg); } while (0)
-#define _W(fmt, arg...)		do { SLOGW(fmt, ##arg); } while (0)
-#define _E(fmt, arg...)		do { SLOGE(fmt, ##arg); } while (0)
+#define AGENT_LOG_TAG "ml-agent"
+
+#define LOG_V(prio, tag, fmt, arg...) \
+  ({ do { \
+    dlog_print(prio, tag, "%s: %s(%d) > " fmt, __MODULE__, __func__, __LINE__, ##arg);\
+  } while (0); })
+
+#define _D(fmt, arg...)		LOG_V(DLOG_DEBUG, AGENT_LOG_TAG, fmt, ##arg)
+#define _I(fmt, arg...)		LOG_V(DLOG_INFO, AGENT_LOG_TAG, fmt, ##arg)
+#define _W(fmt, arg...)		LOG_V(DLOG_WARN, AGENT_LOG_TAG, fmt, ##arg)
+#define _E(fmt, arg...)		LOG_V(DLOG_ERROR, AGENT_LOG_TAG, fmt, ##arg)
+#define _F(fmt, arg...)		LOG_V(DLOG_FATAL, AGENT_LOG_TAG, fmt, ##arg)
```

Comment:
plz add _F for the case of non-tizen (line 42)

Reply:
Thanks, I added as `#define _F g_error`